### ISSUE TITLE
Anderes Icon für blaue Hydranten

### ISSUE THREAD

**Issue by sebkur** [waldbrand/app#17]

**Comment by mozita** [comment]
Die blauen Hydranten können wir als "technischer Hydrant" bezeichnen. Der Erdmann sagt, die blau umrandeten sind die technischen Hydranten.

**Comment by mozita** [comment]
Ich meine das bezogen auf den Editor, wenn man eine neue WES einfügen will, dann gäbe es den technischen Hydrant zur Auswahl.

**Comment by sebkur** [comment]
ja, macht Sinn. Wir müssten rauskriegen, wie man technische Hydranten in OSM mappt. Also gibt's dafür schon ein Tagging-Schema oder müssen wir eins erfinden.
Es gibt bei OSM diese Seite "taginfo", da kann man zu jedem Tag gucken, mit welchen anderen Tags es bisher so kombiniert wird und wie oft: https://taginfo.openstreetmap.org/tags/emergency=fire_hydrant#combinations

**Comment by sebkur** [comment]
tatsächlich gibt es die mit `colour=red/yellow/green/orange/blue` getaggt sind, aber ich könnte mir vorstellen, das bezieht sich eher auf Überflurhydranten und deren Farbe als auf die Schilder bei den Unterflurhydranten

**Comment by mozita** [comment]
Im wiki steht:
Hydrantenschilder mit blauem Rand bedeuten, dass dieser Hydrant neben der Wasserentnahme für Feuerlöschzwecke noch weitere Funktionen erfüllt (z. B. Be- und Entlüften der Leitung). Er kann ganz normal als Hydrant eingetragen werden.

Hydranten mit Hydrantenschildern mit grünem Rand sind Abwasserhydranten und dürfen von der Feuerwehr nicht verwendet werden. Sie sollten daher auch nicht als Hydrant eingetragen werden! 

https://wiki.openstreetmap.org/wiki/DE:Tag:emergency=fire_hydrant
